frontend/src/scenes/feature-flags/featureFlagLogic.tsx

```diff
@@ -273,6 +299,30 @@ export const featureFlagLogic = kea<featureFlagLogicType>({
                 variants.every(({ rollout_percentage }) => rollout_percentage >= 0 && rollout_percentage <= 100) &&
                 variantRolloutSum === 100,
         ],
+        aggregationTargetName: [
+            (s) => [s.featureFlag, s.groupTypes],
+            (featureFlag, groupTypes): string => {
+                if (featureFlag && featureFlag.filters.aggregation_group_type_index != null && groupTypes.length > 0) {
+                    const groupType = groupTypes[featureFlag.filters.aggregation_group_type_index]
+                    return `${groupType.group_type}(s)`
```

Comment:
non-blocking

What's the reason for the () here? Couldn't we just have `s` with ()?

Reply:
It's consistent how we're naming it through the app. Reasoning for `(s)` is that it's a more 'secure' way to pluralize something provided by the user, though not an ideal option.